src/LoaderController.res

```diff
@@ -257,8 +264,9 @@ let make = (~children, ~paymentMode, ~setIntegrateErrorError, ~logger) => {
                   })
                 }
               }
-
-              logger.setLogInfo(~value=Window.href, ~eventName=APP_RENDERED, ())
+              setLaunchTime(_ => dict->Utils.getFloat("launchTime", 0.0))
+              let latency = Date.now() -. launchTime
```

Comment:
What is the difference between the two latencies and why are there two different way to calculate latency?

`let latency = Date.now() -. launchTime`
`let latency = Date.now() -. initTimestamp`

Reply:
Date.now() -. launchTime is total render time (including fetching the iframe dom and other assets over network)
Date.now() -. initTimestamp is the iframe render time only